src/Likvido.Web/Services/Security/IWebSecurityService.cs

```diff
@@ -0,0 +1,10 @@
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Likvido.Web.Services.IP;
+
+[PublicAPI]
+public interface IWebSecurityService
```

Comment:
I think the name of this service is a bit too generic.. maybe we can limit the scope by naming it something like `IRedirectSecurityService` or something like that?

Reply:
Agree, changed it

Reply:
Got the commit names all wrong lol. Still refreshing this from vacation